assets/airports/egll.json

```diff
@@ -997,117 +1014,456 @@
   },
   "departures": {
     "airlines": [
-      ["aal/long", 17 ],
-      ["aca/long", 18 ],
-      ["afl",      7  ],
-      ["afr",      6  ],
-      ["aua",      5  ],
-      ["ava/long", 1  ],
-      ["aza",      5  ],
-      ["baw",      335],
-      ["ces",      2  ],
-      ["cpa",      9  ],
-      ["csn",      1  ],
-      ["dal",      9  ],
-      ["dlh",      30 ],
-      ["ein",      10 ],
-      ["gwi",      8  ],
-      ["klm",      8  ],
-      ["amx/long", 2  ],
-      ["cca/long", 2  ],
-      ["kal/long", 6  ],
-      ["sas",      26 ],
-      ["uae",      5  ],
-      ["ual",      17 ],
-      ["tam",      3  ],
-      ["thy",      6  ],
-      ["vir",      50 ]
+      ["aee",        2],
+      ["ein",        4],
+      ["afl",        1],
+      ["amx/long",   1],
+      ["aca",        7],
+      ["cca",        1],
+      ["afr",        1],
+      ["aic",        4],
+      ["amc",        1],
+      ["asl",        1],
+      ["aza",        2],
+      ["ana",        1],
+      ["aal/long",   8],
+      ["aar",        1],
+      ["aua",        1],
+      ["ava/long",   1],
+      ["baw",      123],
+      ["bel",        1],
+      ["lzb",        1],
+      ["cpa",        1],
+      ["ces",        1],
+      ["csn",        1],
+      ["ctn",        3],
+      ["dal",       14],
+      ["ely",        1],
+      ["uae",        1],
+      ["etd",        1],
+      ["eva",        2],
+      ["ewg",        2],
+      ["gia",        1],
+      ["gwi",        3],
+      ["ibe",        1],
+      ["ibs",        4],
+      ["jal",        1],
+      ["jai",        2],
+      ["klm",        1],
+      ["klc",        1],
+      ["kal/long",   1],
+      ["tam",        1],
+      ["lot",        1],
+      ["dlh",        1],
+      ["mas",        1],
+      ["qfa",        3],
+      ["qtr",        1],
+      ["rja",        1],
+      ["sas",        4],
+      ["sia",        1],
+      ["swr",        1],
+      ["sht",       65],
+      ["tap",        1],
+      ["rot",        1],
+      ["tar",        1],
+      ["thy",        1],
+      ["ual",        6],
+      ["vir",       17],
+      ["vlg",        2]
     ],
     "destinations": [
       "CPT", "BPK", "WOBUN", "DET", "GOGSI", "MAY", "MID"
     ],
     "type": "random",
     "offset": 0,
-    "frequency": 40
+    "frequency": 44
+  },
+  "stars": {
+    "_comment": "In case the BIG  VOR is out of service",
```

Comment:
@indianbhaji For future reference, please keep any comments like this within the star entry, as in `"BIG": {"_comment": "words words words", "icao": "BIG", ...}` due to some recent changes. :smiley: